File: MyCalendar/app/src/main/java/com/sesong/mycalendar/Weather/WeatherItem.java
package com.sesong.mycalendar.Weather;

public class WeatherItem {
    private String category;
    private int fcstValue;

    public WeatherItem(String category, int fcstValue) {
        this.category = category;
        this.fcstValue = fcstValue;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public int getFcstValue() {
        return fcstValue;
    }

    public void setFcstValue(int fcstValue) {
        this.fcstValue = fcstValue;
    }

    @Override
    public String toString() {
        return "WeatherItem{" +
                "category='" + category + '\'' +
                ", fcstValue='" + fcstValue + '\'' +
                '}';
    }
}

File: MyCalendar/app/src/main/java/com/sesong/mycalendar/weather/WeatherItem.kt
package com.sesong.mycalendar.weather

class WeatherItem(var category: String, var fcstValue: Int) {

    override fun toString(): String {
        return "WeatherItem{" +
                "category='" + category + '\'' +
                ", fcstValue='" + fcstValue + '\'' +
                '}'
    }

}